autoload/neomake/makers/ft/javascript.vim

```diff
@@ -36,10 +36,23 @@ endfunction
 
 function! neomake#makers#ft#javascript#standard()
     return {
-        \ 'errorformat': '  %f:%l:%c: %m'
+        \ 'errorformat': '  %f:%l:%c: %m',
+        \ 'postprocess': function('neomake#makers#ft#javascript#StandardEntryProcess')
         \ }
 endfunction
 
+function! neomake#makers#ft#javascript#semistandard()
+    return {
+        \ 'errorformat': '  %f:%l:%c: %m',
+        \ 'postprocess': function('neomake#makers#ft#javascript#StandardEntryProcess')
+        \ }
+endfunction
+
+function! neomake#makers#ft#javascript#StandardEntryProcess(entry)
+    " (Semi)Standard entries are warnings
+    let a:entry.type = 'W'
+endfunction
+
```

Comment:
without this semistandard, and standard makers didn't mark entries as warnings for me